alerts/openshift-container-storage-operator/CephMdsCacheUsageHigh.md

```diff
@@ -27,8 +27,7 @@ Read more about ceph MDS cache configuration [here](https://docs.ceph.com/en/lat
 
 ## Mitigation
 
-Make sure we have enough memory provisioned for MDS cache. Default is 4GB, but recomended
-is minimum 8GB.
+Make sure we have enough memory provisioned for MDS cache.
```

Comment:
Who suggested this? Isn't this more confusing, what is the meaning of enough memory from a customer perspective?

Reply:
In 4.15, OCS operator has introduced profiles :
Lean - 2 Gi
Balanced (default): 6 Gi
Performance: 8 Gi

So keeping a specific value here might be confusing if the user is using a specific profile. 

@agarwal-mudit 

It was discussed a call with the QE and we agreed that it can be removed.  (cc @keesturam)